docs/Reference/API-Methods.md

```diff
@@ -2348,8 +2351,7 @@ The `eth_estimateGas` call does not send a transaction. You must call
 
 #### Parameters
 
-The transaction call object parameters are the same as those for [`eth_call`](#eth_call) except for the
-[`strict` parameter](API-Objects.md#transaction-call-object). If `strict` is set to `true`, the sender
+If you set the [`strict` parameter](API-Objects.md#transaction-call-object) in the transaction call object to `true`, the sender
```

Comment:
I think completely removing all the sentence may make sense as the object is exactly the same as eth_call now.